packages/flutter_tools/lib/src/commands/packages.dart

```diff
@@ -55,7 +53,7 @@ class PackagesCommand extends FlutterCommand {
   String get category => FlutterCommandCategory.project;
 
   @override
-  Future<FlutterCommandResult> runCommand() async => null;
+  Future<FlutterCommandResult> runCommand() async => FlutterCommandResult.success();
```

Comment:
Same question here as in the other PR. If this command doesn't make sense without a subcommand, then should  `runCommand()` return `FlutterCommandResult.fail()`?

Reply:
That makes more sense.